AngularJS/blogs.md

```diff
@@ -0,0 +1,3 @@
+Name | Link
+------------ | -------------
+A better way to learn AngularJS | thinkster.io/a-better-way-to-learn-angularjs
```

Comment:
this is not a clickable link. Requesting you to include a complete link so it would be easier for our repository users to visit shared URLs.